code/src/pages/cart/components/CartItem.scss

```diff
@@ -0,0 +1,66 @@
+@import "@design-system";
+
+.spa-cart-item {
+  display: flex;
+  align-items: center;
+  padding: spacing(4) 0;
+  border-bottom: 1px solid var(--spa-cart-item-border-color, $gray-light);
+
+  &__image {
+    width: spacing(25);
+    margin-right: spacing(5);
+  }
+
+  &__details {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+
+  &__title {
+    font-size: spacing(5);
+    margin-bottom: spacing(3);
+  }

```

Comment:
Use typography variant for size

Reply:
> Use typography variant for size

<img width="536" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/b43131a3-5479-40b3-9318-893b5ab18530">

Already exist `variant` please check more deeply
`h3` variant setted. But I need less `font-size` for it. After changing other variants it's crashed my styles like `subtitle` etc. I made decision to rewrite it

Reply:
Subtitle2 uses just 20px font-sizre. How can it crash your styles?

Reply:
> Subtitle2 uses just 20px font-size. How can it crash your styles?

Rewrited